web/src/features/fbaCalculator/components/FormalFBATable.tsx

```diff
@@ -0,0 +1,35 @@
+import { Table, TableBody, TableCell, TableContainer, TableRow } from '@material-ui/core'
+import { FireCenter } from 'api/fbaAPI'
+import { sortBy } from 'lodash'
+import React from 'react'
+
+interface FormalFBATableProps {
+  testId?: string
+  className: string
+  fireCenter: FireCenter
+}
+
+const buildAdvisoryHeader = (fireCenter: FireCenter) => (
+  <h2>{fireCenter.name}: Fire Behaviour Advisory Summary</h2>
+)
+
+const FormalFBATable = (props: FormalFBATableProps) => {
```

Comment:
I'm not a big fan of the name of this component, but until we figure out specific names for the 2 FBA-related apps, I'll go with this